package.json

```diff
@@ -21,23 +36,24 @@
     "lib"
   ],
   "dependencies": {
-    "babel-runtime": "^6.11.6",
-    "google-auth-library": "^0.9.8",
-    "googleapis": "^12.4.0",
+    "babel-runtime": "^6.26.0",
+    "google-auth-library": "^1.1.0",
+    "googleapis": "^25.0.0",
```

Comment:
This was breaking (most of) the tests. In version 25.x the google API is not returning an array, but instead an object with .data with the payload.

Reply:
Will fix in the next commit.

Reply:
Here is the page describing the change: https://github.com/google/google-api-nodejs-client/blob/474bed78474516ecbb3774f42712063c847f86e8/MIGRATING.md